resources/views/print/modify.blade.php

```diff
@@ -1,6 +1,11 @@
 <div class="card">
     <div class="card-header">@lang('print.modify')</div>
     <div class="card-body">
-       
+        <form method="POST" action="/print/modify_balance">
```

Comment:
Should use [route ](https://laravel.com/docs/5.8/routing#named-routes) function with the named route to get action URL.

Reply:
Done.